### DISCUSSION TITLE
Importing Sionna

### DISCUSSION THREAD

**Discussion by VCK5000** [NVlabs/sionna#319] [Q&A] [answered]
Hi Experts:
I installed Sionna in my PC and after I input import Sionna, I got following error:
jit_cuda_api_init(): could not find symbol "cuDevicePrimaryCtxRelease_v2" -- disabling CUDA backend!

How can I solve this problem? 

Thanks

**Comment by merlinND** [comment]
Hello @VCK5000,

Could you please give us more information about your system?

- OS and OS version (if you are one Windows, are you using WSL?)
- Presence of an NVIDIA GPU or not, and if so, which NVIDIA driver version is installed
- Python version
- Sionna version
- Using Docker or not

**Reply by VCK5000** [reply]
Hi:

Thanks for your reply. Below is my system information:
Windows 10
I don't have an NVIDIA GPU in my laptop. (My laptop has one graphics card: GeForce MX110 )
Python 3.10
Sionna v0.16.1
No docker

Thanks

**Reply by merlinND** [reply, marked as answer]
Thank you, and what is your NVIDIA driver version? Could you try updating it to the latest version?

**Reply by VCK5000** [reply]
Hi:
I updated my driver to the latest version 546.65 and it works.

Thank you very much

**Reply by merlinND** [reply]
Glad to hear it, thank you for reporting back!

**Reply by HamzaBahsir** [reply]
Hello @merlinND , I am having same problem , here is my environment detail
OS and OS version -> Windows 11 Pro and I am not using any WSL
I have Nvidia GPU GeForce GT 730, NVIDIA-SMI 385.54                 Driver Version: 385.54  
Python 3.10
sionna  0.19.1
No docker

**Reply by merlinND** [reply]
Hello @HamzaBahsir,

It looks like you are running a very old NVIDIA driver version, I don't think that it is supported. Please try and update to the most recent version supported by your system.

**Reply by HamzaBahsir** [reply]
Thanks @merlinND , I will try that

**Reply by HamzaBahsir** [reply]
@merlinND , I have tried two driver versions supposedly compatible with GeForce GT 730 but both are not compatible with my system/hardware 474.44-desktop-win10-win11-64bit-international-whql, 475.14-desktop-win10-win11-64bit-international-dch-whql

**Reply by merlinND** [reply]
Hello @HamzaBahsir,

I'm sorry, I don't have a similar system that I could test with. It may be that the GPU or driver isn't supported to use the CUDA backend.
However, you should in principle still be able to use the LLVM (i.e., CPU) backend.
